src/Utilities/for_testing/NativeInitializerPrint.hpp

```diff
@@ -95,7 +95,10 @@ inline std::ostream& operator<<(std::ostream& out, const NativePrint<Vector<T>>&
   out << "{ ";
   auto vec = np_vec.get_obj();
   for (T& t : vec)
-    out << std::setprecision(10) << t << ", ";
+    if constexpr (IsComplex_t<T>::value)
+      out << std::setprecision(10) << '{' << t.real() << ", " << t.imag() << "}, ";
+    else
+      out << std::setprecision(10) << t << ", ";
```

Comment:
I don't quite understand this change. `<<` does support `std::complex`.
The added code doesn't compile if `T` is not `std::complex` because IIRC constexpr still checks syntax, and a simple T=double doesn't support `real()/imag()`.

Reply:
I was wrong. Template is special

"If a constexpr if statement appears inside a [templated entity](https://en.cppreference.com/w/cpp/language/templates#Templated_entity), and if condition is not [value-dependent](https://en.cppreference.com/w/cpp/language/dependent_name#Value-dependent_expressions) after instantiation, the discarded statement is not instantiated when the enclosing template is instantiated."

https://en.cppreference.com/w/cpp/language/if

Reply:
Why the [default style](https://en.cppreference.com/w/cpp/numeric/complex/operator_ltltgtgt) doesn't work for you?
Could you still add a comment explaining the reason for this specialization?